packages/botonic-cli/src/util/bot-config.ts

```diff
@@ -13,15 +13,29 @@ const NPM_DEPTH_0 = 0
 // Also get alpha and beta versions
 const versionRegex = /@botonic\/[^@]+@(\d+\.\d+\.\d+(-[a-zA-Z]+\.\d+)?)/
 
+interface BuildInfo {
+  node_version: string
+  npm_version: string
+  botonic_cli_version: string
+}
+
 type BotonicDependencies = Record<string, { version: string }>
 
+interface Tool {
+  name: string
+  description: string
+}
+
+interface Webview {
+  name: string
+}
+
 export interface BotConfigJSON {
-  build_info: {
-    node_version: string
-    npm_version: string
-    botonic_cli_version: string
-  }
+  build_info: BuildInfo
   packages: BotonicDependencies
+  tools?: Tool[]
+  payloads?: string[]
+  webviews?: Webview[]
```

Comment:
good! but just a question, are these values used along with tools, or are more generic so webviews and payloads can be used also for future configs?
also, should we name tools like `agent_tools` or similar so it's clearer when we read the bot config? or can we use an object to define this like... `agents_config?: {tools, etc.}`